libraries/src/AWS.Lambda.Powertools.Idempotency/Persistence/BasePersistenceStore.cs

```diff
@@ -262,15 +261,11 @@ private string GetHashedPayload(JsonDocument data)
             return "";
         }
         
```

Comment:
This is the only change in Idempotency. Instead of using DevLab.JmesPath we are now using AWS.Lambda.Powertools.JMESPath. The functionality is the same
- Parse the JMESPath expression from the payload
- Generate a hash from the value that was found using the Transform function
- Return the hash to use has a key in DynamoDB